apps/language_server/lib/language_server/providers/code_lens/test.ex

```diff
@@ -53,7 +53,8 @@ defmodule ElixirLS.LanguageServer.Providers.CodeLens.Test do
       %{
         "filePath" => file_path,
         "testName" => block.name,
-        "projectDir" => project_dir
+        "projectDir" => project_dir,
+        "module" => Atom.to_string(block.module) |> String.replace("Elixir.", "")
```

Comment:
nit, commit to the pipeline:
```suggestion
        "module" => block.module |> Atom.to_string() |> String.replace("Elixir.", "")
```

Reply:
conversely, you can also do this via: 
```suggestion
        "module" => block.module |> Module.split() |> Enum.join(".")
```